### PR TITLE
Updating docs with correct links to repo

### PR DESCRIPTION
**Pull request by michaelallen1996**
Updating documentation to correctly show process for adding/editing chapters to the docs.

### CODE REVIEW

File: docs/getting-started/introduction.md [hunk 1]
```diff
@@ -23,8 +23,8 @@ One page summary of how to start a new ArtemisCloud project. [Quick Start →]({
 
 You can contribute to these docs by sending a Pull Request to the [activemq-artemis-operator](https://github.com/artemiscloud/activemq-artemis-operator) Github repository.
 Each chapter is a single markdown file found under the [docs](https://github.com/artemiscloud/activemq-artemis-operator/tree/main/docs) directory.
-Simply update an existing chapter or add a new file for a new one. If you add a new on you should update the [\_index.md](https://github.com/artemiscloud/activemq-artemis-operator/blob/main/docs/_index.md)
-file with the new chapter .
+Simply update an existing chapter or add a new file for a new one. If you add a new one you should create the [\_index.md](https://github.com/artemiscloud/activemq-artemis-operator/blob/main/docs/getting-started/_index.md)
+file within the new chapter .
```
Comment (brusdev) [inline docs/getting-started/introduction.md:27]: Actually, after the [docs refactoring](https://github.com/artemiscloud/activemq-artemis-operator/commit/2b99b39064b5ddb3e3a89f9b78c4f7eef5d19494) this step is not anymore required, it should be removed.
```suggestion
```

Comment (brusdev) [reply docs/getting-started/introduction.md:27]: @michaelallen1996 did you get a chance to review my previous comment?

Comment (michaelallen1996) [reply docs/getting-started/introduction.md:27]: Thanks for the review @brusdev. I'll update the PR to remove those lines entirely.

Comment (brusdev) [reply docs/getting-started/introduction.md]: @michaelallen1996 thanks for you contribution. I squashed your commits and I pushed them to the main branch.

**Pull request closed without merging**